stories/demos/components/Editors/SelectEditor.tsx

```diff
@@ -9,7 +9,7 @@ interface SelectEditorProps {
   menuPortalTarget: Element;
 }
 
-export function SelectEditor<R>({ value, onChange, options, rowHeight, menuPortalTarget }: SelectEditorProps) {
+export function SelectEditor({ value, onChange, options, rowHeight, menuPortalTarget }: SelectEditorProps) {
```

Comment:
Is there an eslint rule to catch unused types?

Reply:
Yes, `@typescript-eslint/no-unused-vars` caught this one.